src/i18n/locales/en/translation.json

```diff
@@ -297,20 +331,26 @@
         "label": "Invoice, Lightning Address or LNURL",
         "placeholder": "Paste invoice, lnurl or lightning address"
       },
-      "submit": {
-        "label": "Continue"
-      },
       "lnurlpay": {
-        "amount_label": "Amount",
-        "comment_label": "Comment",
-        "name_label": "Name",
-        "email_label": "Email",
-        "placeholder": "optional",
-        "confirm_button": "Confirm"
+        "amount": {
+          "label": "Amount (Satoshi)"
```

Comment:
This should be `Amount`. Then the unit-tests will pass.  
I think after this we can switch every other case of "Amount (Satoshi)" to "Amount" and use it via common.